docs/public-networks/get-started/connect/sync-node.md

```diff
@@ -222,6 +250,34 @@ The easiest solution in this scenario is to restart fast sync to obtain a new pi
 
 :::
 
+## Sync mode comparison
+
+| Sync Mode                 | Description         | Requirements            | Method                         | Limitations    |
+|---------------------------|---------------------|-------------------------|--------------------------------|----------------|
+| Snap Synchronization      
+| Recommended for fastest sync and lowest storage requirements on Mainnet.       
+| Besu version 22.4.0 or later | Downloads as many leaves of the trie as possible, reconstructs the trie locally.
+| Cannot switch from fast sync to snap sync mid-process. 
+|
+| Checkpoint Synchronization
+| Efficient sync from a specific checkpoint block configured in the genesis file. 
+| Besu version 22.4.3 or later 
+| Syncs from a checkpoint block defined in the genesis file. 
+| Not supported for QBFT or IBFT 2.0 networks without a checkpoint configuration.
+|
+| Fast Synchronization      
+| Default for named networks except the dev development network.                 
+| None                         
+| Downloads block headers and transaction receipts, verifies chain from genesis block. 
```

Comment:
There's two types of data being synced: chain data (blocks), and world state data.
Snap/checkpoint sync chain data the same way as fast sync. The main difference is how they handle world state data.